Misc/NEWS

```diff
@@ -46,7 +46,7 @@ Core and Builtins
 
 Library
 -------
-
+  
```

Comment:
The blurb entry looks good, thanks, but you also need to remove the change to Misc/NEWS here.